changes/bug31088

```diff
@@ -0,0 +1,5 @@
+  o Minor bugfixes (ipv6):
+    - We check for private IPv6 address alongside their IPv4 equivilants
+      when authorities check descriptors and when relays extend. Previously,
+      we only checked for private IPv4 addresses in those places. Fixes bug
+      31088; bugfix on 0.4.1.1-alpha. Patch by Neel Chauhan.
```

Comment:
Hi Neel, please stop putting the current version in changes files.

Instead, find the first version that implemented IPv6 support or ExtendAllowPrivateAddresses, and make it a bugfix on that version.